.clang-format

```diff
@@ -18,6 +17,7 @@ Cpp11BracedListStyle: 'true'
 IncludeBlocks: Regroup
 IndentCaseLabels: 'true'
 IndentWidth: '4'
+Language: Cpp
```

Comment:
nit: Not sure how/why this got moved but I think `Language` should go at the top of the section.

Reply:
Fixed